tests/ClientBuilderTest.php

```diff
@@ -35,7 +35,7 @@ public function testCreate(): void
 
     public function testHttpTransportIsUsedWhenServeIsConfigured(): void
     {
-        $clientBuilder = new ClientBuilder(['dsn' => 'http://public:secret@example.com/sentry/1']);
+        $clientBuilder = new ClientBuilder(new Options(['dsn' => 'http://public:secret@example.com/sentry/1']));
```

Comment:
To simplify the tests I would use the `create` method so that we avoid creating the `Options` object everywhere

Reply:
I replaced it whenever possible due to the presence of the DSN only. Everywhere else, it may be better to have setters, which are easier to find if needed.